classes/category/maps/Schema.php

```diff
@@ -73,15 +77,40 @@ public function summarizeMany(Enumerable $collection): Enumerable
     protected function mapByProperties(array $props, Category $category): array
     {
         $output = [];
-
+        $context = Application::get()->getRequest()->getContext();
         foreach ($props as $prop) {
             switch ($prop) {
+                case 'subCategories':
+                    $subCategories = Repo::category()->getCollector()
+                        ->filterByParentIds([$category->getId()])
+                        ->filterByContextIds([$context->getId()])
+                        ->getMany();
+
+                    if ($subCategories->isNotEmpty()) {
+                        $output['subCategories'] = $this->mapMany($subCategories)->values();
```

Comment:
This is the sad part of such structure, hopefully users won't have huge nested datasets.
Anyway, I think here it's also needed to protect against circular references.